test/client/components/CommentForm.test.js

```diff
@@ -0,0 +1,71 @@
+require('../support')
+import { mocks, helpers } from '../../support'
+import ConnectedPostList from '../../../src/containers/ConnectedPostList'
+import {
+  findRenderedDOMComponentWithClass,
+  findRenderedDOMComponentWithTag,
+  renderIntoDocument,
+  Simulate
+} from 'react-addons-test-utils'
+const { click, change } = Simulate
+
+describe('CommentForm', () => {
+  describe('in ConnectedPostList', () => {
+    let community = {id: '1', name: 'Foomunity', slug: 'foo'}
+
+    let currentUser = {
+      id: 'x',
+      name: 'Mr. X',
+      avatar_url: '/img/mrx.png'
+    }
+
+    let state = {
+      communities: {
+        [community.slug]: community
+      },
+      people: {
+        current: currentUser
+      },
+      posts: {
+        '1': {
+          id: '1', name: 'post', type: 'offer',
+          user: {id: 1, name: 'jo', avatar_url: ''},
+          communities: [community],
+          description: '<p>A long enough description that the show more button is displayed.</p>' +
+            '<p>A long enough description that the show more button is displayed.</p>' +
+            '<p>A long enough description that the show more button is displayed.</p>' +
+            '<p>A long enough description that the show more button is displayed.</p>' +
+            '<p>A long enough description that the show more button is displayed.</p>' +
+            '<p>A long enough description that the show more button is displayed.</p>'
+        }
+      },
+      totalPostsByQuery: {'subject=community&id=foo': 1},
+      postsByQuery: {'subject=community&id=foo': [1]},
+      pending: {}
+    }
+
+    it.skip('retains comment data after blur', () => {
```

Comment:
I left this in because it took me a long time to write ;) and it might be salvageable once we set up the browser testing